ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -179,6 +182,16 @@ void RSkComponentTextInput::onHandleKey(rnsKey eventKeyType, bool keyRepeat, boo
   textInputMetrics.contentOffset.y = frame.origin.y;
   if (isInEditingMode_ == false && eventKeyType == RNS_KEY_Select ) {
       requestForEditingMode();
+    /*
+       TODO:Launching with default configuration for now.
+            Need to parse & pass properties :
+         1. keyboardType [oskType]
+         2. returnKeyType [returnKeyLabel]
+         3. keyboardAppearance [oskTheme]
+         4. enablesReturnKeyAutomatically [enablesReturnKeyAutomatically]
+         5. placeholder [placeHolderName]
+    */
+    OnScreenKeyboard::launch();
```

Comment:
Do we need OSK launch here ? since requestForEditingMode is already doing it

Reply:
Not Needed with latest code base. I can Remove from Here